### ISSUE TITLE
In Semantic UI React, NVDA/JAWS is not announcing the options in the "Multiple Search Selection" dropdown. 

### ISSUE THREAD

**Issue by msftedad** [Semantic-Org/Semantic-UI#7125]
### Title (Put in field above)
In Semantic UI React, Narrator/NVDA/JAWS is not announcing the options in the "Multiple Search Selection" dropdown. 

### Steps

1. Hit the URL: [Dropdown - Semantic UI React (semantic-ui.com)](https://react.semantic-ui.com/modules/dropdown/#types-dropdown)
2. Tab till "Multiple search Selection" dropdown
3. Check whether In Semantic UI React, Narrator/NVDA/JAWS is announcing the options in the "Multiple Search Selection" dropdown or not.

### Expected Result
In Semantic UI React, Narrator/NVDA/JAWS should announce the options in the "Multiple Search Selection" dropdown.

### Actual Result
In Semantic UI React, Narrator/NVDA/JAWS is not announcing the options in the "Multiple Search Selection" dropdown.

### Version
3.0.0

### Testcase
[Dropdown - Semantic UI React (semantic-ui.com)](https://react.semantic-ui.com/modules/dropdown/#types-dropdown)

**Comment by msftedad** [comment]
Hi team, Any update on this issue?

**Comment by msftedad** [comment]
Hi team, Any update on this issue?

**Comment by msftedad** [comment]
Hi Team, As issue is still repro. Could you please provide update if there is any?
